src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVService/CVImageCriteriaRequest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Text;
+using System.Threading;
+using RegressionGames.StateRecorder.JsonConverters;
+using UnityEngine;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models.CVSerice
```

Comment:
This may require a few other changes
```suggestion
namespace RegressionGames.StateRecorder.BotSegments.Models.CVService
```

Reply:
resolved